Add NTP and salt configuration dialogs for Kubic system roles

thkukuk opened the pull request:
Add NTP configuration dialog for microos system role and ntp + salt configuration for Kubic system roles.

thkukuk committed:
Add MicroOS NTP Configuration Dialog [bsc#1135342]

thkukuk committed:
Version 4.3.0 - Create dialog for Kubic system roles with salt minion

jsrain commented on package/yast2-caasp.spec:
```diff
@@ -17,7 +17,7 @@
 
 Name:           yast2-caasp
-Version:        4.2.0
+Version:        4.3.0
```
Any reason for 4.3.0? All YaST packages targeted for Leap 15.2 or SLE15-SP2 will have 4.2.x, unless there is a reason, please, keep consistent.

thkukuk replied:
The package will only be used in Tumbleweed (not Leap, not SLE15). I used 4.3.0 as this are major new functionality and not only some bug fixes

jsrain reviewed (changes requested):
Please, add a comment to the almost duplicated files about the duplication so that - if someone edits them in the future - he knows that it should be changed at both locations.

thkukuk committed:
Add a comment, that the kubic_*_finish.rb files are identical and
needs always changed at the same time
Follow yast version numbering schema (4.2.1)
